doc/TRIAGING.md

```diff
@@ -0,0 +1,146 @@
+# Triage
+
+## Preface
+
+This document elaborates on the triaging process that we use to prioritize and resolve incoming requests, bug reports etc.
+
+Triaging can be defined as
+
+> In software engineering, triaging is a process where each bug is prioritized on its severity, frequency, risk, impact on the team, etc. The goal of triage is to evaluate, prioritize, and assign the resolution of bugs or defects. The main task the team needs to do is validate the severity of the defect, assign the required resources to solve the defects, make changes as needed, and finalize the resolution of the defects.
+
+quoting [this](https://str-wibowo98.medium.com/what-is-software-triage-b66c6b0a2777) article.
+
+## Why
+
+Before we start, why even bother with triage? It's another thing to learn and spend time on, why do we consider this a worthwhile effort investment? Here are a couple of reasons:
+
+- We care about our users. There isn't much point in developing something and then ignoring the feedback of people for whom you developed the product in the first place. Satisfaction of our users is the most important indicator that we do well, hence we always need to keep user experience in mind and work towards improving it, as otherwise people will go elsewhere.
+- There have been numerous times that our users not only reported important findings, but also fully investigated a potential issue and proposed a reasonable way forward (check out [#5135](https://github.com/input-output-hk/plutus/issues/5135) for a great example). We don't want to miss out on those by not handling reported issues properly.
+
+As for why we need a process -- it's simple: because otherwise we'd revert back to shortly glossing over an issue and moving on hoping it'll somehow resolve itself. Believe it or not, but this strategy wasn't working well at all and we could easily forget about an important report or even miss it completely in the past.
```

Comment:
Perhaps using irony in this document does not set the right tone.

Reply:
Right, I'll fix it, thank you.